tests/src/e2e.rs

```diff
@@ -837,12 +859,13 @@ mod tests {
         while index < n_of_peers {
             let node_path = path.join(format!("anoma-{}", index));
 
-            let mut config = Config::default();
-
-            let _ledger_config = Ledger {
-                tendermint: node_path.join("tendermint").to_path_buf(),
-                db: node_path.join("db").to_path_buf(),
-                ..Default::default()
+            let mut config = Config {
```

Comment:
@Fraccaman I've updated this to use the ledger config with the temp dir, otherwise when running locally, the ledger used in `match_intent` test was picking up state from my local node (the default `.anoma` dir)